app/javascript/channels/index.js

```diff
@@ -3,3 +3,5 @@
 
 const channels = require.context('.', true, /_channel\.js$/)
 channels.keys().forEach(channels)
+
+// import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
Reminder that you've added the `bootstrap` gem and will need to resolve this comment.